### PR TITLE
SDK-1934: Add support for requesting an IDV session be performed usin…

### PR DESCRIPTION
**Pull request by MrBurtyyy**

### CODE REVIEW

File: yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/IbvOptions.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    }
+
+    public static class Builder {
+
+        private String support;
+        private String guidanceUrl;
+
+        private Builder() {}
+
+        public Builder withNotAllowedSupport() {
```
Comment (bucky-boy) [inline yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/IbvOptions.java:35]: Does it read better to say _withIbvNotAllowed_?

File: yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/IbvOptions.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            return withSupport(DocScanConstants.NOT_ALLOWED);
+        }
+
+        public Builder withMandatorySupport() {
```
Comment (bucky-boy) [inline yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/IbvOptions.java:39]: _withIbvMandatory_?

File: yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/IbvOptions.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            return this;
+        }
+
+        public Builder withGuidanceUrl(String guidanceUrl) {
+            this.guidanceUrl = guidanceUrl;
+            return this;
+        }
+
+        public IbvOptions build() {
+            Validation.notNullOrEmpty(support, "support");
```
Comment (bucky-boy) [inline yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/IbvOptions.java:54]: We don't need to enforce this here

File: yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/IbvOptions.java (hunk 1 above)
Comment (MrBurtyyy) [reply yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/IbvOptions.java]: Fixed

File: yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/IbvOptions.java (hunk 2 above)
Comment (MrBurtyyy) [reply yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/IbvOptions.java]: Fixed

File: yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/IbvOptions.java (hunk 3 above)
Comment (MrBurtyyy) [reply yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/IbvOptions.java]: Fixed

**Review by bucky-boy — APPROVED** [review]

**Pull request merged**